README.md

```diff
@@ -41,7 +39,13 @@ How to Install
 6. At Octave prompt, type `syms x`, then `f = (sin(x/2))^3`,
    `diff(f,x)`, etc.
 
+How to install on Ubuntu
+-------------------------
 
+1. If you've not installed Octave yet, use `sudo apt-get install octave`.
+2. Fetch the Debian package liboctave-dev to get the mkoctfile command. Use `sudo apt-get install liboctave-dev` in terminal.
+3. If Python and SymPy not already installed, use `sudo apt-get install python` && `sudo apt-get install python-sympy`.
```

Comment:
Minor nit: this can be simplified to `sudo apt-get install python-sympy`, the python package is pulled in automatically.